test/jit/test_schema_check.py

```diff
@@ -16,6 +18,50 @@
                        "\tpython test/test_jit.py TESTNAME\n\n"
                        "instead.")
 
+# This TorchDispatchTensor Subclass is used to test incorrectly aliasing ops
+# This is done by aliasing the output to the first argument for all ops in INCORRECT_OPS
+
+class IncorrectAliasTensor(torch.Tensor):
```

Comment:
awesome! This could be pretty useful in the future too if we get around to fixing batch_norm (in which case we would need to find a new way to test mutation)